{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -72,9 +75,11 @@ isort = "^5.10.1"
 jupyterlab = "^3.2.5"
 {%- endif %}
 mypy = "^0.931"
+{%- if not cookiecutter.with_fastapi_api|int and not cookiecutter.with_streamlit_app|int %}
+poethepoet = "^0.12.2"
```

Comment:
Could you put this one directly under `pep8-naming` so that we maintain the alphabetical ordering of dev dependencies? Thanks!

Reply:
Done.